tests/test_file_utils.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+import requests
+from transformers.file_utils import CONFIG_NAME, WEIGHTS_NAME, filename_to_url, get_from_cache, hf_bucket_url
+from transformers.testing_utils import DUMMY_UNKWOWN_IDENTIFIER
+
+
+MODEL_ID = DUMMY_UNKWOWN_IDENTIFIER
```

Comment:
I don't follow why asking for `DUMMY_UNKWOWN_IDENTIFIER` in `test_standard_object` sometimes works, or what `PINNED_SHA1, PINNED_SHA256` represent. Comments would help.

I might also test the errors raised for some bad/invalid `revision=` kwargs.

Reply:
Good point, will add some comments/tests

Reply:
This is way better, LGTM.

The thing that had me most confused is that 
[`DUMMY_UNKNOWN_IDENTIFIER`](https://huggingface.co/julien-c/dummy-unknown) actually exists. That constant should probably be named `VALID_DUMMY_IDENTIFIER` or `TINY_ROBERTA`.

Reply:
Yes that's the original tests-only tiny model on the platform!